src/components/Explore/components/FooterLinks/FooterLinks.js

```diff
@@ -0,0 +1,37 @@
+import "./footerLinks.css";
+
+const FooterLinks = () => {
+  return (
+    <div className="footer-links">
+      <a className="footer-links__link" href="/terms-of-service">
```

Comment:
In React, for internal routing, you should use the Link component from react-router-dom instead of the <a> tag. The Link component is specifically designed for client-side navigation and helps prevent full page reloads, providing a smoother user experience.

```
import { Link } from 'react-router-dom';
<Link to="/profile">Go to Profile</Link>
```

Also, we can add a general style for the links